t/command/help_and_version.t

```diff
@@ -29,13 +29,17 @@ subtest 'get version' => sub {
     my ( $output, $stderr, $exit ) = capture { Statocles->run( '--version' ) };
     is $exit, 0;
     ok !$stderr, 'stderr is empty' or diag "STDERR: $stderr";
-    is $output, "Statocles version 1.00 (Perl $^V)\n";
+    my $expected = <<EOF;
+Statocles version 1.00 (Perl $^V)
+Locale: en_US.UTF-8
```

Comment:
Will this pass if the user is in a different locale?

Reply:
And be careful: There is a marginal, but non-zero, chance they don't have the locale we want. I suspect that has been some of the CPAN Testers errors in some of the tests that already work with locales (so they do the appropriate check themselves).

Reply:
I believe the trade-off is to silently get locale-based stuff wrong, as now, or have it (more) visibly fail and have the chance to do something about it. If iteration is necessary, then so be it?

Reply:
Sounds good to me. That's why there's all the locale reporting everywhere ;)